great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -232,6 +232,54 @@ class BaseDataContext(EphemeralDataContext, ConfigPeer):
     GE_YML = "great_expectations.yml"  # TODO: migrate this to FileDataContext. Still needed by DataContext
     GE_EDIT_NOTEBOOK_DIR = GE_UNCOMMITTED_DIR
     DOLLAR_SIGN_ESCAPE_STRING = r"\$"
+    TEST_YAML_CONFIG_SUPPORTED_STORE_TYPES = [
```

Comment:
@lccasagrande I am not sure where these changes came from -- I will be happy to review again once the latest `develop` has been merged into the present branch.  Thank you.

Reply:
Done